core/src/env/engine/mod.rs

```diff
@@ -17,11 +17,19 @@ use crate::env::backend::{
     TypedEnv,
 };
 use cfg_if::cfg_if;
+use core::cell::RefCell;
+use std::thread::LocalKey;
```

Comment:
The std lib import is probably an issue, can you think of another way to get to `LocalKey`?